13_time_series/13_time_series.jl

```diff
@@ -795,7 +795,14 @@ md"Well, good! We started this chapter not knowing how to tackle time series for
 
 As a final summary, when dealing with a time series it is very important to be able to define if it has any latent variables such as trend or seasonality. Once we can find that underlying information, we will be able to generate forecasts with confidence. We just need to look deeper.
 
-### Bibliography
+### Summary
+
+In this chapter we have learned the basic foundations of time series analysis. 
+We have defined what a time series is and delve into a particular method: The exponential smoothing. 
+After building up an intuition of how the simple exponential smoothing works, we continued to introduce more complex versions of the method as the various problems we set ourselves required it. 
+The simple, trended and seasonality methods were presented and coded, generating in that way a much greater understanding of how they work.
```

Comment:
The last two sentences don't seem to include a lot of information, remember to refresh concepts instead of just narrate what happened